src/main/kotlin/no/nav/faktureringskomponenten/controller/FakturaserieController.kt

```diff
@@ -61,6 +63,25 @@ class FakturaserieController @Autowired constructor(
         return ResponseEntity.ok(NyFakturaserieResponseDto(referanse))
     }
 
+    @ProtectedWithClaims(issuer = "aad", claimMap = ["roles=faktureringskomponenten-skriv"])
+    @PutMapping
```

Comment:
Bør være en egen sti for dette.

Reply:
[deb1f1f](https://github.com/navikt/faktureringskomponenten/pull/104/commits/deb1f1f6780c91536962e913f96dee61ca08ff8f)